docs/TrafficSplitting.md

```diff
@@ -0,0 +1,166 @@
+# Traffic Splitting
+
+kn can help you control which revisions get traffic in your Knative Service.
+
+A Knative Service has a traffic mapping, which is a mapping of revisions of the Service to percent amounts of traffic, along with the ability to create unique URLs for particular revisions. It also has the ability to assign traffic to the latest Revision, which can change as new revisions are created.
+
+## Operations
+
+`kn` supports the following traffic operations on traffic block of a Service as part of `kn service update` command.
+
+### 1. Split Traffic:
+
+Every time you update the Service's template, a new Revision is created with 100% traffic routing to it. With every update to the Configuration of the
```

Comment:
I think a lot of this is detail that is not needed here. Or at least not this early. I would rather show the how to get traffic splitting with kn rather than the details of the mechanism. Reason is that it’s petty much useless information for the user of kn. They just want to know what command to use to achieve a goal. I’d keep it that way

Reply:
We added this keeping new users of Knative in mind. For new users, I think its necessary to build some context.